kclvm/parser/src/lexer/mod.rs

```diff
@@ -242,17 +242,16 @@ impl<'a> Lexer<'a> {
             kclvm_lexer::TokenKind::Minus => {
```

Comment:
You can leave a `todo`, and then process multi-character tokens uniformly by processing TokenStream twice, such as `->` and so on. Ref: https://github.com/rust-lang/rust/blob/master/compiler/rustc_parse/src/lexer/tokentrees.rs#L283